CommandsManual.md

```diff
@@ -1794,9 +1861,9 @@ Updates the specified webhook with a new target URL and/or MIME content-type. Fa
 
 | Command<br />Option<br />Alias | Command Option<br />Full Name(s) |
 | --- | --- |
-| -a | --all, --assigned, --allow-disconnect |
+| -a | --all, --assigned, --allow-disconnect, --bitbucket-srv-addr |
 | -ac | --account |
-| -b | --build, --builds |
+| -b | --build, --builds, --bitbucket-srv-config |
```

Comment:
I'm confused by this one... (and the `-e` flag). If I specify the `-b` flag, which command does it actually infer, `--build`, `--builds`, or `--bitbucket-srv-config`.

Reply:
It depends on the command you are invoking. For example, if you write `impt build copy -b xxx`, then it means `--build`. But if you write `impt test bitbucket-server -b xxx`, then it means `--bitbucket-srv-config`.
Aliases are made the way they don't interfere with each other. I.e., there are no commands where both `--build` and `--bitbucket-srv-config` can be used.